src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static deleteDeadAnimal(animal) {
+    this.alive = this.alive.filter(name => name !== animal);
```

Comment:
but it's an array of objects, can name be an object?
just remove animals with health less or equal 0, why to compare objects?